cmd/verify.go

```diff
@@ -47,17 +48,27 @@ var debugCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		var validateOpts *server.VerifySnpOpts
-		if len(teeNonce) != 0 {
-			validateOpts = &server.VerifySnpOpts{
-				Validation:   server.SevSnpDefaultValidateOpts(teeNonce),
-				Verification: &sv.Options{},
+
+		var validateOpts interface{}
+		switch attestation.GetTeeAttestation().(type) {
+		case *pb.Attestation_TdxAttestation:
+			validateOpts = &server.VerifyTdxOpts{
+				Verification: tv.DefaultOptions(),
```

Comment:
Do we not have a teeNonce check for TDX?

Reply:
go-tdx-guest/validate is not in MVP, so we didn't integrate validate in this PR.
And validate package have the teeNonce check .